src/assembler.cpp

```diff
@@ -1242,6 +1242,9 @@ int main(int argc, char* argv[])
                         constant.b = (uint32_t)values[0] != 0;
                         break;
                     }
+
+                    default:
```

Comment:
-1 if this cannot ever happen anyway.

If it can realistically happen, why not properly handle the error?